xla/service/gpu/model/hlo_op_profiles_data.h

```diff
@@ -22,331 +22,961 @@ namespace gpu {
 // The data below is obtained with
 // xla/service/gpu/model:hlo_op_profiler_run
 
-constexpr char kDeviceHloOpProfiles[] = R"pb(
```

Comment:
They are meant to address this error in MSVC:
```
error C2026: string too big, trailing characters truncated
```
`clang-format` made the diff massive but it's just breaking it down to a few concatenated strings as opposed to a single very long one.